chapter8.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml/DTD/xhtml1-transitional.dtd">
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict/dtd">
+<?xml version="1.0" ?>
+
+<!-- start of introduction -->
+  <h1>Currtent Exhibitions</h1>
+  <h2>Olafur Eliasson</h2>
+  <!-- end of introduction -->
+  <!-- start of main text -->
+  <p>Olafur Eliasson was born in copenhagen, Denmark in 1967 to Icelandic parents.</p>
+  <p>He isknown for sculptures and large-scale installation art employing elemental materials such as light, water, and air temperature to enhance the viewer's experience.</p>
+  <!--end of main text -->
+  <!-- <a href="mailto:info@example.org">Contact</a> -->
+
+  <p>Water and air. So very commonplace are these substances, they hardly attract attention - and yet they vouchsafe our very existence.</p>
+  <p id="pullquote">Every time I view the sea I feel a calming sense of sevurity, as if visiting my ancestral home; I embark on a voyage of seeing.</p>
+  <p>Mystery of mysteries, water and air are right there before us in the sea.</p>
+
+  <p class="important">For a one-year period from November 2010, the marugame Genichiro-Inokuma Museum of Contemporary Art (MIMOCA) will host a cycle of four Hiroshi Sugimoto exhibitons.</p>
+  <p>Each will showcase works by the artist thematically contextualized under the headings "Science," "Architecture," "History" and "Religion" so as to present a comprehensive panorama of the artist's oeuvre.</p>
+  <p class="important admittance">Hours:10:00 - 18:00 (No admittance after 17:30)</p>
+  <h1>Hiroshi Sugimoto</h1>
+  <p>The dates for the ORIGIN OF ART exhibition are as follows:</p>
+  <ul>
+    <li>Science: 21 Nov - 20 Feb 2010/11</li>
+    <li>Architecture: 6 Mar - 15 May 2011</li>
+    <li>History: 29 May - 21 Aug 2011</li>
+    <li>Religion: 28 Aug - 6 Nov 2011</li>
+  </ul>
+
+  Timed to a single revolution of the planet around the sun at a 23.4 degrees tilt that plays out the rhythem of the seasons, this <em>Origins of Art</em> cycle is organized around four themes: <b>science, architecture, history</b> and <b>religion</b>
+
+  <div id="header">
+    <img src="images/logo.gif" alt="Anish Kapor" />
+    <ul>
+      <li><a href="index.html">Home</a></li>
+      <li><a href="biography.html">Biography</a></li>
+      <li><a href="works.html">Works</a></li>
+      <li><a href="contact.html">Contact</a></li>
+    </ul>
+  </div><!-- end of header -->
+
+  <p>Anish Kapoor won the Turner Prize in 1991 and exhibitied at teh <span class="gallery">Tate Modern</span> gallery in London in 2003.</p>
+
+  <iframe width="450" height="350" src="http://maps.google.co.uk/maps?q=moma+new+york&amp;output=embed"></iframe>
+
+  <iframe src="http://maps.google.co.uk/maps?q=moma+new+yourk&amp;output=embed" width="450" height="350" frameborder="0" scrolling="no"> </iframe>
+
+  <!Doctype html>
```

Comment:
It's cool you putting all this into one file, as long as you understand that in real life you CANT put all these into one file. It might be better to separate them out into multiple chapter 7 files like we talked about. 

Remember you can only have one html tag per document, one body, one head, etc.